.ado/continuous.yml

```diff
@@ -8,27 +8,6 @@ variables:
   - group: platform-override-zero-permission-token
 
 stages:
-  - stage: Setup
-    jobs:
-      - job: Setup
-        variables:
-          - template: variables/windows.yml
-        pool:
-          vmImage: windows-2019
```

Comment:
You change this to ubuntu in the new setup.yml, will it still work?

Reply:
Yep. It's the same as PR setup.